rca/static_src/javascript/programmes/components/ProgrammesCategories/CategoryItem.scss

```diff
@@ -0,0 +1,106 @@
+.category-item {
+    $padding-top-small: $gutter;
+    $padding-top-medium: $gutter * 2.5;
+    $padding-top-large: $gutter * 1.5;
+    $root: &;
+    display: grid;
+    grid-column: 1 / span 2;
+    grid-template-columns: 1fr;
+    padding-top: $gutter;
+    color: inherit;
+    font-family: $font--primary;
+    border-left: 1px solid $color--grid-line-dark;
+
+    // Bottom border, done with a pseudo element rather than a border,
+    // so it can overflow to the right and leave some space for people to scroll
+    // without accidentally clicking a link.
+    &::after {
+        content: '';
+        grid-column: 1 / span 2;
+        height: 1px;
+        width: calc(100% + #{$gutter});
+        background-color: $color--grid-line-dark;
+
+        @include media-query(medium) {
+            width: calc(100% + #{$gutter * 3});
+        }
+
+        @include media-query(large) {
+            grid-column: 1 / span 4;
+        }
+    }
+
+    @include media-query(medium) {
+        grid-template-columns: 2fr 3fr;
```

Comment:
This is one of the places where somehow the layout doesn’t follow the grid.

Reply:
I see what you mean in terms of tablet. Should it follow the grid for desktop or are they happy to follow tablet usage.

Design:
![Screenshot 2020-01-08 at 15 18 37](https://user-images.githubusercontent.com/31627284/71989966-82a3a780-322a-11ea-9e6d-904917832578.png)

Site:
![Screenshot 2020-01-08 at 15 18 49](https://user-images.githubusercontent.com/31627284/71989992-8afbe280-322a-11ea-8e02-f5f900c9e5dc.png)

Reply:
@chris-lawton this is part of the design – your screenshot is of the desktop "large" layout